build.sh

```diff
@@ -0,0 +1,175 @@
+#! /bin/bash
+: '
+For building Calamares using this program, in the application "Terminal" type:
+"/pathToThisFolder/build.sh" "[arguments]"
```

Comment:
But this does not just build Calamares (which would be `ninja -C build` for me in my development setup), it does other things as well.

Reply:
Checking dependencies is part of building.